db/migrate/20230405092923_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
```

Comment:
- [ ] Since we are adding `devise` to an already existing user model, the appropriate syntax would be adding column e.g. `add_column :users :email, :string` instead of directly adding `t.string :email, ` as we are not creating a new table we are changing an exiting one. This form of migration would raise an active record error as these fields don't exist yet. I would highly suggest you to change the syntax in order for the migration to work properly.